pkg/statdb/statdb.go

```diff
@@ -284,27 +286,28 @@ func (s *Server) UpdateBatch(ctx context.Context, updateBatchReq *pb.UpdateBatch
 // CreateEntryIfNotExists creates a statdb node entry and saves to statdb if it didn't already exist
 func (s *Server) CreateEntryIfNotExists(ctx context.Context, createIfReq *pb.CreateEntryIfNotExistsRequest) (resp *pb.CreateEntryIfNotExistsResponse, err error) {
 	APIKeyBytes := createIfReq.APIKey
+
 	getReq := &pb.GetRequest{
 		NodeId: createIfReq.Node.NodeId,
 		APIKey: APIKeyBytes,
 	}
 	getRes, err := s.Get(ctx, getReq)
-	if err != nil {
+	if err != nil && strings.Contains(err.Error(), "no rows in result set") {
```

Comment:
maybe use [`errs.Class`'s `.wrap`/`.has` method(s)](https://godoc.org/github.com/zeebo/errs#Class.Has) for this? [(example)](https://github.com/zeebo/errs#error-classes)

Reply:
^I'm interested in seeing how this turns out 😮